test/quantization.cpp

```diff
@@ -431,7 +432,7 @@ TEST_CASE(op_capture)
         auto pb = mm->add_parameter("b", s2);
         auto pc = mm->add_parameter("c", s2);
         auto pa = mm->add_instruction(migraphx::make_op("add"), p1, p2);
-        auto ps = mm->add_instruction(migraphx::make_op("dot"), pa, pb, pc);
+        auto ps = migraphx::add_apply_alpha_beta<float>(*mm, {pa, pb, pc}, "dot", 1, 1);
```

Comment:
`<float>` can be removed? It should deduce the type from the template.

Reply:
Done